dowhy/causal_estimator.py

```diff
@@ -126,6 +182,48 @@ def do(self, x):
     def construct_symbolic_estimator(self, estimand):
         raise NotImplementedError
 
+    def get_confidence_intervals(self, estimate):
```

Comment:
Can consider re-introducing certain parameters here that may need to be reused. For example, consider a usecase where a person wants both 90% and 95% CIs, or they want to try different sample sizes.
In this case, it is better to have optional parameters in the method signature that override the self. parameters. See the first two lines of the test_significance method for an example.

Actually, that makes me think: do we have an easy way for a user to access the estimator object? If not, it may be worthwhile returning a reference to it in the CausalEstimate object. Now that we are doing computationally intensive methods, it makes sense that a user can compute CIs without having to run the full estimator again.

Reply:
So both sample_size_fraction and num_ci_simulations can be added as optional parameters here.

Reply:
That makes sense. I'll make that change.